modules/Using gem5/_posts/2000-05-01-models-cpu.md

```diff
@@ -6,6 +6,71 @@ livestream_code: cDv-g-c0XCY
 example_code: /materials/using-gem5/05-cpu-models
 ---
 
-- Use different gem5 CPU models (Timing Simple, Atomic, O3, Minor, Trace, etc.)
+This is a summary document for the gem5 bootcamp session on CPU models.
+This tutorial covers following topics:
 
-- Trace with debug flags and talk about start/end
+- gem5 CPU models
+- Using gem5 CPU models
+- Understanding the CPU model differences via an experiment
+
+**Note:** This document is meant to complement the slides used in the bootcamp and will refer to the slides for detailed instructions on how to run the experiments of this tutorial.
+
+## gem5 CPU Models
+
+Following picture provides an overview of the differences among gem5 CPU models:
+
+![Summary of gem5 CPU Models](../../../assets/img/gem5CPUs.png)
```

Comment:
I don't know if this kind of relatively linking will work, but it's more stable to do "/assets/img/gem5CPUs.png". This file (2000-05-01-models-cpu.md) could be moved at some point and doing it this way guarantees the link does not break.

Reply:
Fixed.